MigrateServicePreferenceFromLegacy/function.json

```
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "queueName": "%MIGRATE_SERVICES_PREFERENCES_PROFILE_QUEUE_NAME%",
+      "connection": "QueueStorageConnection",
```

The new queue has been created on a new storage account. Env variable nale is FN_APP_STORAGE_CONNECTION_STRING

```suggestion
      "connection": "FN_APP_STORAGE_CONNECTION_STRING",
```